File: C/Chapter 3 - Control Statement/2K221010/patteren_alphabet2_ascending.c
//Loops:

//Pattern Programs

/*

Ex. Alphabet Pattern 2 - Ascending

A
B B
C C C
D D D D
E E E E E

*/

#include<stdio.h>
void main()
{
    int count,i,j;

    printf("Enter the count: ");
    scanf("%d",&count);
    printf("\n");

    for(i=1;i<=count;i++){
        for(j=1;j<=i;j++){
            printf("%c ",i+64);
        }
        printf("\n");
    }
}

File: Back End/Python/Chapter 4 - Control Statements/2K221229/patteren_alphabet2_ascending.py
# //Loops:
#
# //Pattern Programs
#
# /*
#
# Ex. Alphabet Pattern 2 - Ascending
#
# A
# B B
# C C C
# D D D D
# E E E E E
#
# */

count = int(input("Enter the count: "))

for i in range(0, count + 1, 1):
    for j in range(0, i, 1):
        print(chr(i + 64), end=' ')
    print(end='\n')
